docs/source/new-docs/getting-started/system-requirements/index.rst

```diff
@@ -3,30 +3,71 @@
-     - This is a Reference page - please consult `this guide <https://docs.divio.com/documentation-system/reference/>`_
++------------------+---------------------------------------------------------+
+| Operating System | Minimum Version                                         |
++==================+=========================================================+
+| Linux            | RedHat 8, AlmaLinux 8, RockyLinux 8 equivalent or later |
++------------------+---------------------------------------------------------+
+|                  | Debian 10 equivalent or later                           |
++------------------+---------------------------------------------------------+
+| MacOS            | 12 or later *(not officially supported)*                |
++------------------+---------------------------------------------------------+
+| Windows          | 10 or later *(not officially supported)*                |
++------------------+---------------------------------------------------------+
 
- - A PR from your feature branch to ``main`` 😊
++----------+-------------+-------------+--------------------------------------+
+| Hardware | Minimum     | Recommended | Supports                             |
++==========+=============+=============+======================================+
+| CPU      | 2 Cores     | 4 Cores     | A single concurrent GeoIPS process   |
++----------+-------------+-------------+--------------------------------------+
+|| Memory  || 12GB       || 128GB      || Production of most GeoIPS imagery.  |
+||         ||            ||            || Some Geostationary imagery requires |
+||         ||            ||            || > 90GB RAM.                         |
++----------+-------------+-------------+--------------------------------------+
+| Storage  | 20GB on SSD | 20GB on SSD | Base GeoIPS system installation.     |
++----------+-------------+-------------+--------------------------------------+
+||         || 20GB       || As needed  || Additional disk space for input and |
+||         ||            ||            || output data storage. May be SDD,    |
+||         ||            ||            || HDD, or NAS.                        |
++----------+-------------+-------------+--------------------------------------+
+
+Developer System Requirements
+-----------------------------
+
+In addition to the required disk space for users, developers will require:
+
++----------+-------------+-------------+---------------------------+
+| Hardware | Minimum     | Recommended | Supports                  |
++==========+=============+=============+===========================+
+| Storage  | 40GB        | 40GB        | Integration test datasets |
++----------+-------------+-------------+---------------------------+
+|          | 20GB on SSD | 20GB on SSD | Static cartopy shapefiles |
+|          |             |             | for integration tests.    |
++----------+-------------+-------------+---------------------------+
+
+Future System Requirements
```

Comment:
These aren't really future system requirements per se. They're more-so recommendations on how to build/purchase a system that will be able to run future versions of GeoIPS well.

Reply:
I will update this to say "If you are speccing out a system for an operational environment, please reach out to the GeoIPS team to discuss system needs and the impact of our future development plans on your system." This should link to the contact page.